src/com/ivoninsky/task1/BattleShip.java

```diff
@@ -0,0 +1,147 @@
+package com.ivoninsky.task1;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class BattleShip {
+    private int[][] field;
+    private Random random = new Random();
+    private List list;
+
+    public BattleShip() {
+        this.field = new int[10][10];
+        this.list = new ArrayList<>();
+    }
+
+    public void printField() {
+        for (int i = 0; i < field.length; i++) {
+            for (int j = 0; j < field.length; j++) {
+                if (field[i][j] == 0 || field[i][j] == 2) {
+                    System.out.print("  ");
+                } else {
+                    System.out.print("[]");
+                }
+            }
+            System.out.println();
+        }
+    }
+
+    public void printField2() {
+        for (int i = 0; i < field.length; i++) {
+            for (int j = 0; j < field.length; j++) {
+                System.out.print(field[i][j] + " ");
+            }
+            System.out.println();
+        }
+    }
+
+    public int[][] getField() {
```

Comment:
it's breaking incapsulation, remove this method
you have method for printing so you don't need to expose your field outside the class